lib/ui/blocs/posts_list/posts_list_state.dart

```diff
@@ -67,11 +67,16 @@ class PostsLoaded extends PostsListState {
     );
   }
 
-  /// gets all errored post
+  /// gets all errored posts
   List<Post> get getErroredPosts => posts
```

Comment:
Since this is already a getter (due to the `get` keyword before its name), I think it's better to rename it simply `erroredPosts`